### PR TITLE
Add links from experimental methods to the helper functions that allow access to them.

### PR DESCRIPTION
**Pull request by tylerbenson**
Otherwise it's not easy to discover them.

We could also consider putting a link the other direction. That would allow IDE's to find the reference easier. I chose to skip that for now.

### CODE REVIEW

**Comment by jkwatson** [comment]
needs the formatter run on it...

**Comment by tylerbenson** [comment]
Not sure why it's failing now.

File: sdk/logs/src/main/java/io/opentelemetry/sdk/logs/SdkLoggerProviderBuilder.java [hunk 1]
```diff
@@ -108,7 +109,10 @@ public SdkLoggerProviderBuilder setClock(Clock clock) {
   }
 
   /**
-   * Set the logger configurator, which computes {@link LoggerConfig} for each {@link
+   * This method is experimental so not public. You may reflectively call it using {@link
+   * SdkLoggerProviderUtil#setLoggerConfigurator(SdkLoggerProviderBuilder, ScopeConfigurator)}
+   *
+   * <p>Set the logger configurator, which computes {@link LoggerConfig} for each {@link
    * InstrumentationScopeInfo}.
```
Comment (jack-berg) [inline sdk/logs/src/main/java/io/opentelemetry/sdk/logs/SdkLoggerProviderBuilder.java:116]: These are flipped. By convention, the first sentence of javadoc is a short summary of method. This new text provides supplementary information, which is important but secondary to the summary sentence.

**Comment by jack-berg** [comment]
Build will pass if you merge main into this branch. I fixed the problem the build is failing with in #6537.

**Review by jack-berg — APPROVED** [review]

**Pull request merged**